.gitignore

```diff
@@ -1,2 +1,8 @@
 node_modules/
 
+# IDE - VSCode
+.vscode/*
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+.history/*
```

Comment:
I'm open to undoing this, but I found that VSCode's `settings.json` file wasn't being used anywhere, and making one locally for my own development purposes didn't (yet) warrant adding it to the repo